.github/workflows/armada-release.yml

```diff
@@ -0,0 +1,88 @@
+name: Release Armada
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  release-armadactl:
+    runs-on: buildjet-4vcpu-ubuntu-2204
```

Comment:
Do we really need to pay money to use these runners for a release process?  What is Perf without these runners?